README.md

```diff
@@ -1,2 +1,44 @@
 # Free Code Camp - Applied InfoSec Challenges
 =============================================
+
+This is the repository for the first 11 freeCodeCamp [Information Security with HelmetJS Challenges](https://learn.freecodecamp.org/information-security-and-quality-assurance/information-security-with-helmetjs/).  
+For the challenges from the 12th on upwards, you have to clone [this one](https://github.com/freeCodeCamp/boilerplate-bcrypt). 
+
+If you haven't worked with **git** before, you might first want to read a few   
+[resources](http://try.github.io/) about it, or go the [Glitch](https://glitch.com/) 
+way as described on the above mentioned challenge introduction page.
+
+To clone this repository, go to a directory of your choice and run:
+```bash
+git clone https://github.com/freeCodeCamp/boilerplate-infosec.git
+cd boilerplate-infosec
+```
+But to be able to import it in Glitch or the coding platform of your choice,  
+you have to [create your own Repository on GitHub](https://help.github.com/articles/creating-a-new-repository/).  
+But before you are able to push to it, you have to remove the remote origin of
+freeCodeCamp,   
+rename the "gomix" branch to "master" and delete the gomix branch
+to have a clutter-free repository:
+```
+git remote remove origin
+git branch -m master
+git branch -d gomix
+```
+Then follow the steps outlined in **...or push an existing repository from the command line**  
+on the **Quick setup** screen, e.g.:
+```
+git remote add origin git@github.com:<yourname>/<your_repository>.git
+git push -u origin master
+```
+
+Now you should have a fully functional Repository you can import to a coding platform.  
+In order to develop locally, there is one more step necessary. Run:
+```
+npm install
```

Comment:
You should use an [inline code-block](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet#code) in this case, since there is only one command to run which is fairly short.

Reply:
Moved this to it's own block on purpose, because it's one of the most important steps and StackOverflow is "cluttered" by questions like "why is {insert dependency} not found".

Reply:
In that case, you could make the code-block even more noticeable by adding `**Important:** ...` to  the beginning of the sentence.

Reply:
#done